doc/source/user_guide/tutorials/index.rst

```diff
@@ -3,94 +3,143 @@
 Tutorials
 ---------
 
+What do you need to know
+++++++++++++++++++++++++
 
-The following tutorials show in details how to use basic features of PyDPF-Core
-for each usage main step.
+This tutorials will guide you through the steps required to access, analyze,
+and transform simulation data using  PyDPF-Core.
 
-:bdg-dark-line:`Define simulation data` :octicon:`arrow-right;1em` :bdg-dark-line:`Store input data in DPF` :octicon:`arrow-right;1em` :bdg-dark-line:`Transform available data` :octicon:`arrow-right;1em` :bdg-dark-line:`Extract data`
+The tutorials are more substantive and complex than examples found in the standard
+documentation set. They are designed to teach how to perform a task and understand
+some concepts, providing detailed explanations at each stage.
 
-**Define input data**
+DPF interacts with results data that have to be stored in some kind  of structure.
+They can be automatically generated from the results file (detailed in the access
+and extract results part ) or generated from scripting.
 
+Overview
+++++++++
 
-.. grid:: 3
+There are four main steps to transform simulation data into output data that can
+be used to visualize and analyze simulation results:
 
-    .. grid-item-card:: Data Sources
-       :link: ref_tutorials_model
-       :link-type: ref
-       :text-align: center
+    :bdg-dark-line:`Define simulation data` :octicon:`arrow-right;1em` :bdg-dark-line:`Store input data in DPF` :octicon:`arrow-right;1em` :bdg-dark-line:`Transform available data` :octicon:`arrow-right;1em` :bdg-dark-line:`Extract data`
 
-    .. grid-item-card:: DPF model
-       :link: ref_tutorials_model
-       :link-type: ref
-       :text-align: center
+The following tutorials will guide through those steps. The tutorials explains separately
+the used methods for files of each acceptable solver types.
 
-**Load data**
 
+.. topic:: Importing and opening results files
```

Comment:
```suggestion
.. topic:: Extract data from result files
```

Reply:
I don't know if using "extract " here is a great idea. Because in the other section we talk about extracting (like if I want to have a displacement result). So here is it actually extract a result?